### ISSUE TITLE
errors everywhere

### ISSUE THREAD

**Issue by BKR-13** [Huntinex/rauton#2]

**Comment by Huntinex** [comment]
Hey !
Can you tell me more about these errors?
I guess this happened because you didn't install all the tools properly!

Cheers,
- HuntineX

**Comment by BKR-13** [comment]
```error: please provide the download path for tools```
Does the download path must be "/usr/bin" or "/bin"... or any random path?

**Comment by Huntinex** [comment]
Yes, the download path must be in the `$PATH` variable, like `/usr/bin`
